routing/routing_tests/checkpoint_predictor_test.cpp

```diff
@@ -0,0 +1,145 @@
+#include "testing/testing.hpp"
+
+#include "routing/checkpoint_predictor.hpp"
+
+#include "geometry/point2d.hpp"
+
+#include <cstdint>
+#include <utility>
+#include <vector>
+
+using namespace std;
+
+namespace
+{
+void TestAlmostEqual(double v1, double v2)
+{
+  double constexpr kEpsMeters = 1.0;
+  TEST(my::AlmostEqualAbs(v1, v2, kEpsMeters), (v1, v2));
```

Comment:
Why? I often write functions with names starting with word "Test" which test something inside them. It's worth using when you don't want to stop tests if one test fails.

In this case the method TestAlmostEqual() let us shorten code.